packages/bbob-core/test/index.test.ts

```diff
@@ -11,15 +11,17 @@ describe('@bbob/core', () => {
     const res = process([], '[style size="15px"]Large Text[/style]');
     const ast = res.tree;
 
-    expect(res.html).toBe('[{"tag":"style","attrs":{"size":"15px"},"content":["Large"," ","Text"]}]');
+    expect(res.html).toBe('[{"tag":"style","attrs":{"size":"15px"},"content":["Large"," ","Text"],"startTagPos":{"start":0,"end":19},"endTagPos":{"start":29,"end":37}}]');
     expect(ast).toBeInstanceOf(Array);
     expect(stringify(ast)).toEqual(stringify([
       {
         tag: 'style',
         attrs: { size: '15px' },
-        content: ["Large", " ", "Text"]
+        content: ["Large", " ", "Text"],
+        startTagPos: { start: 0, end: 19 },
```

Comment:
I think `tag` and `pos` is redundant here. We already know that this is a tag.

Seems `start` looks better.
Also to avoid repeat same words.

```js
start: { from: 0, to: 19 }
end: { from: 29, to: 37 }
```

Mission of this library is - be small as possible. So thats I use one-word names everywhere where possible